docs/start/general/wallet/the-main/the-main.md

```diff
@@ -42,8 +42,8 @@ To discover the address of the particular crypto asset, click on the asset’s i
 :::
   
 
-In the bottom of the wallet window the user would find some complementary commands.
+On the bottom of the wallet window, the user will find some additional commands.
 
-Use “Refresh to find” to manually refresh the main page. “Manage (assets) manually” — from this menu the user can manually add a custom token (entering token route) to the wallet account or manually search for a token.
+On the bottom of the wallet window, the user can find a button labeled "Refresh to find" to manually refresh the main page if the wallet balance or transactions are not updated. Additionally, user may find a "Manage (assets) manually" button which will allow them to manually add a custom token by entering the token address or manually search for a token.
```

Comment:
@adamnewton55 I suppose we should write here "to manually refresh the assets list if the balances are not fetched"